src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -145,18 +145,19 @@ const CourseCalendarEvent = (props: CourseCalendarEventProps) => {
                         <tr>
                             <td className={classes.alignToTop}>Section code</td>
                             <Tooltip title="Click to copy course code" placement="right">
-                                {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */}
```

Comment:
This can be removed since onClick was moved into the button with the section code